### PR TITLE
Implement Iron Mocha Addons stub and replace

### PR DESCRIPTION
**Pull request by cdata**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,3 +21,47 @@ test('can be clicked', function(done) {
...
+
+### `replace`
+
+The `replace` addon allows the tester to replace all usages of one element with
+another element within all Polymer elements created within the time span of the
+test. Usage example:
+
+```javascript
+beforeEach(function() {
+  replace('x-foo').with('x-fake-foo');
```
Comment (justinfagnani) [inline README.md]: Some elements and styling may depend on the tag name. Can we replace just the implementation instead?

Comment (cdata) [inline README.md]: `stub` above replaces "just the implementation." There are cases where you do not care about styling as much as you care about replacing a complex element implementation. Between the two helpers, you'll probably get 95% of the way there.

**Comment by azakus** [comment]
LGTM

**Comment by morethanreal** [comment]
Nice, lgtm.

**Comment by RossRothenstine** [comment]
Is this PR dead? These helpers would be very useful to my development.

**Comment by cdata** [comment]
Nope, not dead. Just needs a little TLC. I'll try to finish it off soon!

**Comment by RossRothenstine** [comment]
Great! I am excited. I am having an issue with using the stub method, but I wanted to verify it was still alive before reporting it.

**Comment by notwaldorf** [comment]
(This has landed separately in WCT)

**Pull request closed without merging**